### PR TITLE
Rebase to alpine 3.17 with php8.1

### PR DESCRIPTION
**Pull request by nemchik**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [ ] I have read the [contributing](https://github.com/linuxserver/docker-xbackbone/blob/main/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,19 +12,19 @@ LABEL maintainer="gilbn"
...
-    php8-pdo_sqlite \
-    php8-gd \
-    php8-json \
-    php8-fileinfo \
-    php8-zip \
-    php8-ftp \
-    php8-ldap \
-    php8-tokenizer \
-    php8-intl && \
+    php81 \
```
Comment (thespad) [inline Dockerfile]: Base already has php81

File: Dockerfile [hunk 2]
```diff
@@ -10,19 +12,19 @@ LABEL maintainer="gilbn"
...
+    php81-sqlite3 \
+    php81-mysqli \
+    php81-pdo_mysql \
+    php81-pdo_sqlite \
+    php81-gd \
+    php81-json \
```
Comment (thespad) [inline Dockerfile]: json is part of the main php81 package now.

**Review by thespad — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (nemchik) [reply Dockerfile]: Resolved in https://github.com/linuxserver/docker-xbackbone/pull/21/commits/4a53c216ab533f905a6f1d80d57c942e298f5b36

File: Dockerfile (hunk 1 above)
Comment (nemchik) [reply Dockerfile]: Resolved in https://github.com/linuxserver/docker-xbackbone/pull/21/commits/ea1c221b59f954a2c4177ee96d2f48ce3e8bc430

**Review by thespad — APPROVED** [review]

**Pull request merged**